Targets/STM32F4/STM32F4_CAN.cpp

```diff
@@ -0,0 +1,1597 @@
+// Copyright Microsoft Corporation
+// Copyright Oberon microsystems, Inc
+// Copyright GHI Electronics, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <algorithm>
+#include <string.h>
+#include "STM32F4.h"
+
+///////////////////////////////////////////////////////////////////////////////
+
+#ifdef INCLUDE_CAN
+
+#define CAN_MESSAGES_MAX 128
```

Comment:
Remove this and replace it with a define in the device.h file, "STM32F4_CAN_RX_BUFFER_DEFAULT_SIZE".